test/wallet-http-test.js

```diff
@@ -85,6 +85,30 @@ for (const witnessOpt of witnessOptions) {
       addr = Address.fromString(str, node.network);
     });
 
+    it('should enable seed phrase recovery', async () => {
+      const options = {
+        passphrase: 'PASSPHRASE',
+        mnemonic: 'zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo wrong'
+      };
+
+      const testwallet = await wclient.createWallet(`test_seed-${witnessOpt}`, options);
+      assert.strictEqual(testwallet.master.encrypted, false);
+
+      const master1 = await wclient.getMaster(`test_seed-${witnessOpt}`);
+      assert.strictEqual(master1.encrypted, false);
+      assert.strictEqual(master1.mnemonic.phrase, 'zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo wrong');
+
+      await wclient.lock(`test_seed-${witnessOpt}`);
+      const master2 = await wclient.getMaster(`test_seed-${witnessOpt}`);
+      assert.strictEqual(master2.encrypted, true);
+      assert.strictEqual(master2.key, undefined);
```

Comment:
I thought we were going to test `mnemonic` is undefined here?

Reply:
I think I lost that commit while rebasing :(